dipy/align/tests/test_transforms.py

```diff
@@ -0,0 +1,233 @@
+from dipy.align.transforms import *
```

Comment:
Please import specific names (best) or import the module (`import dipy.align.transforms as dit`) and use with ``dit.`` namespace.  The `*` makes it difficult to spot bad imports, among other things.